### DISCUSSION TITLE
Can I use RAM to expand VRAM on Linux?

### DISCUSSION THREAD

**Discussion by Threnos** [vladmandic/sdnext#1481] [Q&A] [answered]
I've been searching for a solution for, it seems to be, ages. I've been bottlenecked by my 4Gb vram and often times find that I lack just a couple hundred Mb to perform what I want.
I know that Win drivers 535 started doing this by default, but no luck for linux users.

Linux Mint 20.3 Cinnamon
NVIDIA Corporation GA107M [GeForce RTX 3050 Ti Mobile]

**Comment by vladmandic** [comment]
no way to control it from user-code, either enable or disable.

**Reply by Threnos** [reply]
I found [this](https://askubuntu.com/questions/1273869/on-ubuntu-and-in-hardware-is-it-possible-to-extend-gpu-dram-with-cpu-ram) question pointing to a something called [SpeedTorch](https://discuss.pytorch.org/t/introducing-speedtorch-4x-speed-cpu-gpu-transfer-110x-gpu-cpu-transfer/56147)

**Reply by Threnos** [reply]
And is there a way to enable it?

**Reply by vladmandic** [reply, marked as answer]
that is totally different and not relevant here. like i said, this cannot be done from user-code - torch and nvidia stated so, its not the question has it been done in this repo.

**Comment by Tillerz** [comment]
What vladmantic said, nothing anyone can do except nVidia changing their linux drivers.

**Comment by Threnos** [comment]
Related:
https://forums.developer.nvidia.com/t/non-existent-shared-vram-on-nvidia-linux-drivers/260304
https://github.com/NVIDIA/open-gpu-kernel-modules/issues/663
